docs/advanced-topics/kubernetes-dashboard.md

```diff
@@ -0,0 +1,134 @@
+# Kubernetes Dashboard Access and Authentication
+
+## Default Setup
+
+    kubernetesDashboard:
+      adminPrivileges: true
+      insecureLogin: false
+
+In the default setup the Dashboard is configured using the `--auto-generate-certificates` flag and has Admin privileges
+
+###### Access the dashboard  using `kubectl proxy` command:
+
+    http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/
+
+
+## Expose the Dashboard using a ELB with self-signed certificates
+
+kubernetesDashboard:
+  adminPrivileges: false
+  insecureLogin: false
+
+Ex.
+
+    apiVersion: v1
+    kind: Service
+    metadata:
+      annotations:
+        external-dns.alpha.kubernetes.io/hostname: "kubedash.example.com"
```

Comment:
I guess we are assuming `external-dns` is the default now but might be worth excluding from this example?

Reply:
Yes, I guess it can be excluded. I let it because it won't cause any error if it's specified.

Reply:
Sure, seems fine either way just an opportunity to simplify.